servers/rendering/renderer_rd/shaders/effects/blur_raster.glsl

```diff
@@ -110,26 +116,30 @@ void main() {
 		GLOW_ADD(vec2(0.0, 2.0), 0.122581);
 		GLOW_ADD(vec2(0.0, -1.0), 0.233062);
 		GLOW_ADD(vec2(0.0, -2.0), 0.122581);
-		color *= blur.glow_strength;
+
 		frag_color = color;
 	}
 
 #undef GLOW_ADD
 
 	if (bool(blur.flags & FLAG_GLOW_FIRST_PASS)) {
+		// In the first pass bring back to correct color range else we're applying the wrong threshold
+		// in subsequent passes we can use it as is as we'd just be undoing it right after.
+		frag_color *= blur.luminance_multiplier;
+
 #ifdef GLOW_USE_AUTO_EXPOSURE
 
 		frag_color /= texelFetch(source_auto_exposure, ivec2(0, 0), 0).r / blur.glow_auto_exposure_grey;
 #endif
 		frag_color *= blur.glow_exposure;
 
-		float luminance = max(frag_color.r, max(frag_color.g, frag_color.b));
+		float luminance = dot(frag_color.rgb, vec3(0.299 / 0.587, 1.0, 0.114 / 0.587));
```

Comment:
As discussed last week, lets keep the old luminance calculation. I changed it to this one fairly arbitrarily, but the old calculation appears to better reflect the perceived brightness when colors become saturated. 

The same goes for the compute version, should be changed to ``max(color.r, max(color.g, color.b))``

Reply:
I'll change this back but I think that changing the shader in the clustered renderer should be done in a separate PR with the explanation of why we're doing this as I assume this should also result in the tonemap changes you made being reverted ?

Reply:
After further discussion with @clayjohn, also changed luminance calculation on clustered

Reply:
Mobile:
![image](https://user-images.githubusercontent.com/1945449/167750734-1b478ba3-19a8-49a4-b038-046a26b7ead2.png)

Clustered:
![image](https://user-images.githubusercontent.com/1945449/167750753-d9e46c9a-18e5-4325-8a80-45f540589e1c.png)